From f491b982c4eac4c493104d30ffaf031312d520dc Mon Sep 17 00:00:00 2001
From: lzc256 <i@lzc256.com>
Date: Sat, 20 Jun 2026 18:00:00 +0800
Subject: [PATCH] auth_test.go: add friendsEnabled/acceptInvitesEnabled to
 UpdatePlayer calls

Two test call sites in auth_test.go were missing the two new bool
params added to UpdatePlayer. Pass false,false for both.
---
 auth_test.go | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/auth_test.go b/auth_test.go
index 98603ae..8bd4132 100644
--- a/auth_test.go
+++ b/auth_test.go
@@ -301,7 +301,7 @@ func (ts *TestSuite) testAuthenticateMultipleProfiles(t *testing.T) {
 		firstPlayerName := "FirstPlayer"
 		secondPlayerName := "SecondPlayer"

-		firstPlayer, err := ts.App.UpdatePlayer(&GOD, user.Players[0], &firstPlayerName, nil, nil, nil, nil, false, nil, nil, false)
+		firstPlayer, err := ts.App.UpdatePlayer(&GOD, user.Players[0], &firstPlayerName, nil, nil, nil, nil, false, nil, nil, false, false)
 		assert.Nil(t, err)

 		secondPlayer, err := ts.App.CreatePlayer(&GOD, user.UUID, secondPlayerName, nil, false, nil, nil, nil, nil, nil, nil, nil)
@@ -387,7 +387,7 @@ func (ts *TestSuite) testAuthenticateMultipleProfiles(t *testing.T) {

 		// When the username matches one of the available player names, that
 		// player should automatically become the selectedProfile.
-		_, err = ts.App.UpdatePlayer(&GOD, user.Players[0], Ptr(TEST_USERNAME), nil, nil, nil, nil, false, nil, nil, false)
+		_, err = ts.App.UpdatePlayer(&GOD, user.Players[0], Ptr(TEST_USERNAME), nil, nil, nil, nil, false, nil, nil, false, false)
 		assert.Nil(t, err)

 		rec = ts.PostJSON(t, ts.Server, "/authenticate", authenticatePayload, nil, nil)
--
2.54.0
